vars/runBenchmarkTestScript.groovy

```diff
@@ -71,44 +78,60 @@ void call(Map args = [:]) {
     editBenchmarkConfig("${WORKSPACE}/benchmark.ini")
     String userTags = getMetadataTags(args.userTag.toString(), buildManifest)
```

Comment:
move this inside the `execute-test` if logic. And the above if-else logic as well. 
```
 if (args.insecure.toBoolean()) {
                sleep(5)
            } else {
                sleep(120)
            }
```